pkg/tool/common.go

```diff
@@ -126,3 +128,11 @@ func GetKstTimestamp() time.Time {
 
 	return kst
 }
+
+func GetBaseTime(t time.Time) time.Time {
+	return time.Date(t.Year(), t.Month(), t.Day(), 0, 0, 0, 0, t.Location())
+}
+
+func GetTimePrefix(t time.Time) string {
+	return fmt.Sprintf("%d%d%d", t.Year(), t.Month(), t.Day())
```

Comment:
How about padding with zero for TimePrefix like "%d%02d%02d"?

Reply:
Sounds great :)